packages/@aws-cdk/cdk/test/core/test.jsx.tsx

```diff
@@ -4,11 +4,11 @@ import { Construct, jsx, Root } from '../../lib';
 export = {
     'jsx can be used to create "trees" of constructs'(test: Test) {
         const tree = <MyRoot>
-            <MyConstruct name='child1' prop1='hi' prop2={21} >
-                <MyConstruct name='child11' prop1='there' />
-                <MyConstruct name='child12' prop1='boo' />
+            <MyConstruct id='child1' prop1='hi' prop2={21} >
```

Comment:
That sure is one place where `id` looks *so much better*! 👌🏻

Reply:
Agreed